lib/elixir/lib/calendar.ex

```diff
@@ -65,7 +65,7 @@ defmodule Calendar do
   @typedoc "The time zone ID according to the IANA tz database (e.g. Europe/Zurich)"
   @type time_zone :: String.t()
 
-  @typedoc "The time zone abbreviation (e.g. CET or CEST or BST etc.)"
+  @typedoc "The time zone abbreviation (e.g. CET, CEST, BST, et cetera)."
```

Comment:
I don't think "et cetera" is an improvement. Please let's use simple wording, "and so on" works in every case.

Reply:
Just to clarify the intention of the PR. We never said that the non-native speaker doesn't know the meaning of "et cetera". What we were trying to tackle is that it wouldn't know how to read the contraction, which is not "etc." is many languages.

I will change it anyway.

Reply:
Sorry, I thought it was implied. We have zero gain using "et cetera" instead of "and so on", and I’m all for trying to be as inclusive as possible to people with different English backgrounds, so I'd just go for the simpler language 🙃

Reply:
No worries. All __et cetera__ entries have been replaced.